spec/requests/v1/api_conditional_request_spec.rb

```diff
@@ -132,18 +135,16 @@
 
     context "index actions" do
       let!(:project) { create_list(:project_with_contents, 2, owner: user).first }
-      let(:url) { "/api/projects" }
+      let!(:url) { "/api/projects" }
```

Comment:
Not a big deal but I don't think this needs the '!'. I'm kind of curious if you encountered something that caused you to add it.